--- usr/lib/ubiquity/target-config/30accessibility.old	2011-05-14 07:08:54.593187810 +0200
+++ usr/lib/ubiquity/target-config/30accessibility	2011-05-14 07:30:44.975683714 +0200
@@ -73,9 +73,10 @@
                         fi
 
 			if [ -x /target/usr/bin/orca ]; then
-				if [ -f /home/$USERNAME/.orca/user-settings.py ]; then
-					cp -r /home/$USERNAME/.orca /target/home/$TARGET_USERNAME/
-					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.orca
+				if [ -f /home/$USERNAME/.local/share/orca/user-settings.conf ]; then
+					mkdir -p /target/home/$TARGET_USERNAME/.local/share
+					cp -r /home/$USERNAME/.local/share/orca /target/home/$TARGET_USERNAME/.local/share
+					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.local/share/orca
 				else
 					mkdir -p /target/home/$TARGET_USERNAME/.orca
 					echo "import orca.settings" >> /target/home/$TARGET_USERNAME/.orca/user-settings.py
@@ -103,11 +104,14 @@
                         remove_applet fast_user_switch
                         sed -i -e 's/# Host alias specification/Defaults\tenv_keep = "ORBIT_SOCKETDIR XDG_SESSION_COOKIE GTK_MODULES"\n\n# Host alias specification/g' /target/etc/sudoers
 			if [ -x /target/usr/bin/orca ]; then
-				if [ -f /home/$USERNAME/.orca/user-settings.py ]; then
-					cp -r /home/$USERNAME/.orca /target/home/$TARGET_USERNAME/
-					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.orca
-					cp -r /home/$USERNAME/.orca /target/var/lib/gdm
-					chroot /target chown -R gdm.gdm /var/lib/gdm/.orca
+				if [ -f /home/$USERNAME/.local/share/orca/user-settings.conf ]; then
+					mkdir -p /target/home/$TARGET_USERNAME/.local/share
+					mkdir -p /target/var/lib/gdm/.local/share
+					cp -r /home/$USERNAME/.local/share/orca /target/home/$TARGET_USERNAME/.local/share
+					cp -r /home/$USERNAME/.local/share/orca /target/etc/skel/.local/share
+					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.local/share/orca
+					cp -r /home/$USERNAME/.local/share/orca /target/var/lib/gdm/.local/share
+					chroot /target chown -R gdm.gdm /var/lib/gdm/.local/share/orca
 				else
 					mkdir -p /target/home/$TARGET_USERNAME/.orca
 					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.orca
@@ -124,11 +128,13 @@
                         gct -s -t bool /desktop/gnome/applications/at/visual/startup true
                         gct -s -t string /desktop/gnome/applications/at/visual/exec orca
                         if [ -x /target/usr/bin/orca ]; then
-				if [ -f /home/$USERNAME/.orca/user-settings.py ]; then
-					cp -r /home/$USERNAME/.orca /target/home/$TARGET_USERNAME/
-					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.orca
-					cp -r /home/$USERNAME/.orca /target/var/lib/gdm
-					chroot /target chown -R gdm.gdm /var/lib/gdm/.orca
+				if [ -f /home/$USERNAME/.local/share/orca/user-settings.conf ]; then
+					mkdir /target/home/$TARGET_USERNAME/.local/share
+					mkdir -p /target/var/lib/gdm/.local/share
+					cp -r /home/$USERNAME/.local/share/orca /target/home/$TARGET_USERNAME/.local/share
+					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.local/share/orca
+					cp -r /home/$USERNAME/.local/share/orca /target/var/lib/gdm/.local/share
+					chroot /target chown -R gdm.gdm /var/lib/gdm/.local/share/orca
 				else
 	                                mkdir -p /target/home/$TARGET_USERNAME/.orca
 					echo "import orca.settings" >> /target/home/$TARGET_USERNAME/.orca/user-settings.py
