*** 30accessibility.old	2010-04-15 09:03:12.269953680 +0200
--- 30accessibility	2010-04-15 09:05:37.390952853 +0200
*************** for x in $(cat /proc/cmdline); do
*** 107,112 ****
--- 107,113 ----
  					cp -r /home/$USERNAME/.orca /target/home/$TARGET_USERNAME/
  					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.orca
  					cp -r /home/$USERNAME/.orca /target/var/lib/gdm
+ 					cp -r /home/$USERNAME/.orca /target/etc/skel/
  					chroot /target chown -R gdm.gdm /var/lib/gdm/.orca
  				else
  					mkdir -p /target/home/$TARGET_USERNAME/.orca
*************** for x in $(cat /proc/cmdline); do
*** 128,133 ****
--- 129,135 ----
  					cp -r /home/$USERNAME/.orca /target/home/$TARGET_USERNAME/
  					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.orca
  					cp -r /home/$USERNAME/.orca /target/var/lib/gdm
+ 					cp -r /home/$USERNAME/.orca /target/etc/skel/
  					chroot /target chown -R gdm.gdm /var/lib/gdm/.orca
  				else
  	                                mkdir -p /target/home/$TARGET_USERNAME/.orca
*************** for x in $(cat /proc/cmdline); do
*** 136,141 ****
--- 138,144 ----
  	                                echo "orca.settings.enableBraille = True" >> /target/home/$TARGET_USERNAME/.orca/user-settings.py
  					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.orca
  					cp -r /home/$TARGET_USERNAME/.orca /target/var/lib/gdm
+ 					cp -r /home/$USERNAME/.orca /target/etc/skel/
  					chroot /target chown -R gdm.gdm /var/lib/gdm/.orca
  				fi
                          fi
