--- 30accessibility.old	2010-04-05 04:20:48.003105886 -0400
+++ 30accessibility.new	2010-04-05 04:20:42.072249024 -0400
@@ -75,7 +75,9 @@
 			if [ -x /target/usr/bin/orca ]; then
 				if [ -f /home/$USERNAME/.orca/user-settings.py ]; then
 					cp -r /home/$USERNAME/.orca /target/home/$TARGET_USERNAME/
+					cp -r /home/$USERNAME/.orca /target/var/lib/gdm/
 					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.orca
+					chroot /target chown -R gdm.gdm /target/var/lib/gdm/.orca
 				else
 					mkdir -p /target/home/$TARGET_USERNAME/.orca
 					echo "import orca.settings" >> /target/home/$TARGET_USERNAME/.orca/user-settings.py
@@ -105,7 +107,9 @@
 			if [ -x /target/usr/bin/orca ]; then
 				if [ -f /home/$USERNAME/.orca/user-settings.py ]; then
 					cp -r /home/$USERNAME/.orca /target/home/$TARGET_USERNAME/
+					cp -r /home/$USERNAME/.orca /target/var/lib/gdm/
 					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.orca
+					chroot /target chown -R gdm.gdm /target/var/lib/gdm/.orca
 				else
 					mkdir -p /target/home/$TARGET_USERNAME/.orca
 					chroot /target chown -R $TARGET_USERNAME.$TARGET_USERNAME /home/$TARGET_USERNAME/.orca
