--- /usr/bin/ubiquity.orig	2010-03-26 06:51:35.210739619 -0400
+++ /usr/bin/ubiquity	2010-03-26 06:53:06.729866839 -0400
@@ -66,10 +66,7 @@
                     pass
 
         if frontend == 'gtk_ui':
-            toexec = ['gksudo', '--preserve-env']
-            if desktop:
-                toexec.extend(['--desktop', desktop])
-            toexec.append('--')
+            toexec = ['runsu', '-E']
         elif frontend == 'kde_ui':
             # We need to pass through xauth information.  kdesudo won't do,
             # though, as it can't be made to tell sudo to preserve the
